src/backend/tests/test-data/teams.test-data.ts

```diff
@@ -24,3 +25,17 @@ export const sharedTeam1: SharedTeam = {
   members: [],
   projects: []
 };
+
+export const justiceLeague = {
+  teamId: '1',
+  teamName: 'Justice League',
+  description: 'hiii :3',
+  slackId: '1234',
+  leaderId: 1,
+  leader: { ...batman, googleAuthId: 'a' },
```

Comment:
batman already has google auth id (please take this out everywhere ylou see it idk why people do this every time)